eth/eventhandler/handlers.go

```diff
@@ -25,7 +25,8 @@ import (
 const encryptedKeyLength = 256
 
 var (
-	ErrAlreadyRegistered            = fmt.Errorf("operator registered with the same operator public key")
+	ErrOperatorPubkeyAlreadyExists  = fmt.Errorf("operator public key already exists")
+	ErrOperatorIDAlreadyExists      = fmt.Errorf("operator ID already exists")
```

Comment:
I see that this is the current convention, but wouldn't we want eventually to have these as `error.New()`?

Reply:
@anatollupacescu, [pkg/errors](https://github.com/pkg/errors) is deprecated and archived. We trying to use `fmt.Errorf` istead